Source/Charts/Charts/ChartViewBase.swift

```diff
@@ -562,12 +561,12 @@ open class ChartViewBase: NSUIView, ChartDataProvider, AnimatorDelegate
         {
             if h == nil
```

Comment:
To follow the spirit of your PR, you should flip the order of the if-else, i.e.
```
if let h = h { ... }
else { ... }
```

Reply:
👍